### ISSUE TITLE
[Report]cannot switch to desktop mode if external display is connected

### ISSUE THREAD

**Issue by mindtheegab** [holoiso-eol/holoiso#551]
## Your Issue:
```
cannot switch to desktop mode if external display is connected(official dock)
```

## Your device's full specs
```
steamdeck
```

## Your OS version (`cat /etc/os-release | grep VARIANT_ID | cut -d '"' -f 2`):
```
snapshot20220921.1733

```

## Your installation type:
```
deckperience
```

## Have you updated your installation recently?
```
yes
```

**Comment by mindtheegab** [comment]
device will just restart into deck UI, disconnecting the display and clicking restart to desktop is the workaround

**Comment by Grimish-ng** [comment]
Having this issue as well. Except gamescipe does bot recover or go back to steamis interface. Have to switch to a cli session and force a reboot.

**Comment by suboxide** [comment]
Did a reinstall past week and ended up with the same problem. But somehow it fixed itself.
No idea what happened i have a jsaux dock and kept having the problem even did 2 reinstalls. But I tried my travel hub at the office at random and was like wtf this works. Took hub to home tried and worked perfectly tried jsaux dock again and worked perfectly ever since.
No idea if it was the dock but since using my travel hub my docking got fixed.
